Backpack/src/main/java/net/skyscanner/backpack/calendar/view/WeekdayHeaderView.kt

```diff
@@ -47,6 +47,8 @@ internal class WeekdayHeaderView @JvmOverloads constructor(
 
   internal fun initializeWithLocale(locale: Locale) {
     val formatter = DateTimeFormatter.ofPattern("ccc", locale)
+    val descriptionFormatter = DateTimeFormatter.ofPattern("cccc", locale)
```

Comment:
Why `cccc`?

Reply:
For screenreader it would speak the complete word. eg `Monday` and not `Mon`